scripts/apps/contacts/components/ListTypeIcon.jsx

```diff
@@ -2,10 +2,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export const ListTypeIcon = ({item}) => {
-    let iconClass = item.first_name ? 'icon-user' : 'icon-business';
-
+    const iconClass = item.first_name ? 'icon-user' : 'icon-business';
+    const typeTooltip = item.first_name ? gettext('Person Contact') : gettext('Organisation Contact');
+    
     return (
-        <div key="type-icon" className="list-field type-icon">
+        <div key="type-icon"
+        	className="list-field type-icon"
+        	data-sd-tooltip={item.public ? typeTooltip : typeTooltip + gettext(' (Private)')}
```

Comment:
it's better to use `+ ' ' + gettext('(Private)')` otherwise the space might be truncated via translation